CMakeLists.txt

```diff
@@ -304,13 +305,17 @@ elseif(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
   set(CPACK_RPM_DEMO_PACKAGE_SUMMARY "${CPACK_COMPONENT_DEMO_DISPLAY_NAME}")
   set(CPACK_RPM_DEMO_PACKAGE_ARCHITECTURE noarch)
   set(CPACK_RPM_DEMO_FILE_NAME RPM-DEFAULT)
+  if(NOT GO_SEPARATE_LINUX_PACKAGES)
```

Comment:
If I understand correctly (I'm not aware of packaging processes), is it really needed to keep option to manage separate packages? Why not keeping it simple?

Reply:
I kept this option for possible future integration with OBS and making native packages for various distributions.

Reply:
Ok, understand